### DISCUSSION TITLE
Add a badge like Fresh

### DISCUSSION THREAD

**Discussion by rajrupdasofficial** [coollabsio/coolify#2312] [Feature Requests]
Add a badge like **Fresh** so I can include in my **website** and in my projects **readme** files which are deployed using coolify.
I also can promote this cool service also I can brag about it to my friends and Clients.

Thanks

**Comment by apertureless** [comment]
@rajrupdasofficial 
Yeah I also needed that. Because I have some headless cms + static sites and needed the badge as a status. So people knew, when the generation is done. 

You can get the status from the API. And make your own badge.

I wrote a small service that generates the badge, tho.

https://github.com/nextindex/coolify-badge-service
